modules/mvt/package.json

```diff
@@ -31,6 +31,7 @@
     "build-bundle": "webpack --display=minimal --config ../../scripts/webpack/bundle.js"
   },
   "dependencies": {
+    "earcut": "^2.0.6",
```

Comment:
Nit: As mentioned in my RFC comments, I would have added this to `@loaders.gl/math` to make it a bit more separated.

Reply:
As I wasn't modifying the dependency at all I figured it would be cleaner to include it directly. However after reading your RFC comments (thanks!), and thinking some more I realized that we can indeed improve the perf of the earcut routine by passing in the winding order we have already computed in `classifyRings`, which earcut recomputes. Rather than include this change in this PR, I will do it in a separate PR, to make it easier to review and so this PR can be landed now.